src/components/pages/request-holiday/request-holiday.component.js

```diff
@@ -18,32 +18,43 @@ import Section from '../../commons/section/section.component';
 
 import useStyles from './request-holiday.styles';
 
-const names = [
-  'Tom',
-  'Anna',
-  'Daniel',
-  'Cailey',
-];
+
 const RequestHoliday = () => {
   const classes = useStyles();
+  const [supervisors] = React.useState([
+    'Tom',
+    'Anna',
+    'Daniel',
+    'Cailey',
+  ]);
```

Comment:
We do not need this supervisors to be handled inside the component. We would like to get those as a property with default empty array value.